tests/ZfcRbacTest/Guard/RoutePermissionsGuardTest.php

```diff
@@ -410,7 +410,7 @@ public function testProperlyFillEventOnAuthorization()
 
     public function testProperlySetUnauthorizedAndTriggerEventOnUnauthorization()
     {
-        $eventManager = $this->getMock('Zend\EventManager\EventManagerInterface');
+        $eventManager = $this->getMock('Zend\EventManager\EventManager');
```

Comment:
because `EventManagerInterface` does not have `triggerEvent()` but `EventManager` has it